packages/components/pie-divider/src/index.ts

```diff
@@ -1,18 +1,38 @@
 import { LitElement, html, unsafeCSS } from 'lit';
-
+import { property } from 'lit/decorators.js';
+import { validPropertyValues } from '@justeattakeaway/pie-webc-core';
 import styles from './divider.scss?inline';
-import { DividerProps } from './defs';
+import { DividerProps, variants, orientations } from './defs';
 
 // Valid values available to consumers
 export {
     type DividerProps,
+    variants,
+    orientations,
 };
 
 const componentSelector = 'pie-divider';
 
 export class PieDivider extends LitElement implements DividerProps {
+    @property()
```

Comment:
I _think_ we can/should still set the type to string in the property decorator as seen [here](https://lit.dev/docs/components/properties/#declare-with-decorators)

Reply:
good one! will add